src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -990,7 +990,7 @@ cdef class AioServer:
         # TODO(lidiz) if users create server, and then dealloc it immediately.
         # There is a potential memory leak of created Core server.
         if self._status != AIO_SERVER_STATUS_STOPPED:
-            _LOGGER.warning(
+            _LOGGER.debug(
```

Comment:
:+1: Good catch. I noticed this log as well.

Reply:
Yeah. It is only valuable to power users, but it was annoyingly triggered by `KeyboardInterrupt`.